src/lib/components/Tooltip.svelte

```diff
@@ -102,12 +105,12 @@
     id={idToUse}
     class:noWrap
     class:top
-    class:not-rendered={!notEmptyString(text)}
+    class:not-rendered={isAbsent}
     class:visible={targetIsHovered}
     bind:this={tooltipComponent}
     style={tooltipStyle}
   >
-    {text}
+    {#if nonNullish(text)}{text}{/if}<slot name="tooltip-content" />
```

Comment:
Will it be more predictable to use if else here?

Reply:
I wouldn't expect the user to use both the `text` prop and the slot, but if they do I think it should either show both or cause an error. I don't think we should hide one or the other silently.